### PR TITLE
Add PostgreSQL certificate support

### PR DESCRIPTION
**Pull request by nikicat**
May be it's not so good implementation, but it works (tests are missed)

### CODE REVIEW

File: lib/run-query.js [hunk 1]
```diff
@@ -154,7 +155,10 @@ clients.postgres = function (query, connection, queryResult, callback) {
     password: connection.password,
     database: connection.database,
     host: connection.host,
-    ssl: connection.postgresSsl
+    ssl: connection.postgresSsl ? {
```
Comment (rickbergfalk-uptycs) [inline lib/run-query.js]: I'm not entirely certain, but I think we still need to support ssl: true here. Setting to true without passing in any cert or key will use ssl but not alter any ssl parameters... (I think anyways - I've only set this once a long time ago when querying heroku postgres)

**Comment by rickbergfalk-uptycs** [comment]
thanks @nikicat! I'll happily merge - just one small fix to consider :)

**Comment by nikicat** [comment]
Ok, @rickbergfalk-uptycs, I'll fix it

**Comment by rickbergfalk** [comment]
@nikicat I'm doing some refactoring with the way the connections UI is done... merging this into my dev branch so we don't lose out on your work :)

**Pull request merged**